setup.py

```diff
@@ -1,37 +1,36 @@
 import pathlib
 from setuptools import setup, find_packages
 
-VERSION = "0.3.20"
+VERSION = "1.3.0"
 DESCRIPTION = 'CDOT Work Zone WZDx Translators'
 HERE = pathlib.Path(__file__).parent
 README = (HERE / "README.md").read_text()
 
-# Setting up
 setup(
-    # the name must match the folder name 'verysimplemodule'
-    name="wzdx-translator-jacob6838",
+    name="cdot-wzdx-translator",
     version=VERSION,
-    author="Jacob Frye",
-    author_email="jfrye@neaeraconsulting.com",
+    author="CDOT",
+    author_email="CDOT_rtdh_prod@state.co.us",
```

Comment:
Does this email need to change based on environment?

Reply:
This email is static to the package, and does not change!